packages/angular-material/test/tsconfig.test.json

```diff
@@ -7,5 +7,5 @@
   },
   "include": ["**/*.ts", "**/*.tsx"],
   "exclude": ["node_modules"],
-  "files": ["../src/index.ts"]
+  "files": ["../src/library/index.ts"]
```

Comment:
Minor: I think this can still point to the src/index.ts file, can't it? Probably doesn't make a difference though

Reply:
This file is useless, it's an artifact from previous version. I'll delete it.